### PR TITLE
[13.0][MIG] kpi

### PR DESCRIPTION
**Pull request by flachica**

### CODE REVIEW

**Comment by flachica** [comment]
I think that the odoo.conf of travis is not correct. [base_external_dbsource](https://github.com/OCA/server-backend/tree/13.0/base_external_dbsource) not finded here https://travis-ci.org/github/OCA/reporting-engine/jobs/694682931#L774 because this repo not in [oca_dependencies](https://raw.githubusercontent.com/OCA/reporting-engine/13.0/oca_dependencies.txt). If I PR this file, It be corrected automatically?

**Comment by etobella** [comment]
You must add `server-backend`

**Comment by flachica** [comment]
Added

File: kpi/__manifest__.py [hunk 1]
```diff
@@ -3,12 +3,12 @@
 
 {
     "name": "Key Performance Indicator",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
     "website": "http://www.savoirfairelinux.com",
     "license": "AGPL-3",
     "category": "Report",
-    "depends": ["base_external_dbsource", "web_widget_color",],
```
Comment (etobella) [inline kpi/__manifest__.py]: Why are you removing this dependancy?

File: kpi/models/kpi_threshold_range.py [hunk 2]
```diff
@@ -49,40 +49,29 @@ class KPIThresholdRange(models.Model):
     _name = "kpi.threshold.range"
     _description = "KPI Threshold Range"
 
+    VALUE_TYPES = [
```
Comment (etobella) [inline kpi/models/kpi_threshold_range.py]: if you want to reuse this data, maybe it is better to add to a function model (@api.model) as a hook in order to reuse it

**Review by etobella — CHANGES_REQUESTED** [review]

File: kpi/__manifest__.py (hunk 1 above)
Comment (etobella) [reply kpi/__manifest__.py]: My mistake, color has been added on odoo core :wink:

**Review by etobella — APPROVED** [review]
Tested and reviewed :+1: LGTM

**Review by albariera — APPROVED** [review]
Technical review LGTM :+1:

**Pull request merged**